core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -116,12 +116,12 @@
           MeasurementUnit.create(6, bytes));
 
   // Common histogram bucket boundaries for bytes received/sets DistributionViewDescriptors.
-  private static final List<Double> RPC_BYTES_BUCKET_BOUNDARIES = Arrays.asList(
```

Comment:
They are meant to be constants but that's a reasonable point. I'll change them to be UnmodifiableLists in the next pr.